aisynphys/database/schema/cortical_location.py

```diff
@@ -15,7 +15,10 @@
         ('layer', 'str', 'Name of the layer the cell is in.'),
         ('distance_to_pia', 'float', 'The distance from the cell to the pial surface in m.'),
         ('distance_to_wm', 'float', 'The distance from the cell to the white matter in m.'),
-        ('fractional_depth', 'float', 'The cortical depth of the cell where pia is 0 and wm is 1.')
+        ('fractional_depth', 'float', 'The cortical depth of the cell where pia is 0 and wm is 1.'),
+        ('layer_depth', 'float', 'Absolute depth within the layer in m.'),
+        ('fractional_layer_depth', 'float', 'Fractional depth within the cells layer.'),
+        ('position', 'object', '2D array, position of cell in the slice (in m)'),
```

Comment:
Is this the same as `cell.position` ?

Reply:
nope, i can clarify the description, it's the 2D position in image coordinates. also i realized i never converted units to m for these, is that preferable or could i just change the description to microns?

Reply:
I prefer to have unscaled units wherever possible. However in this case, I'm not sure how the position will be used? If it's just position relative to the DAPI image, then perhaps the position is best left in image pixels?